fdbserver/workloads/ClogTlog.actor.cpp

```diff
@@ -132,14 +132,12 @@ struct ClogTlogWorkload : TestWorkload {
 				// recovery state hasn't changed in 30s, exclude the failed tlog
 				std::vector<StringRef> tokens;
 				state Optional<ConfigureAutoResult> conf;
-				state std::vector<std::string> holder;
 
 				CODE_PROBE(true, "Exclude failed tlog");
 				TraceEvent("ExcludeFailedLog")
 				    .detail("TLog", self->tlog.get())
 				    .detail("RecoveryState", self->dbInfo->get().recoveryState);
-				std::string addr = "exclude=" + formatIpPort(self->tlog.get().ip, self->tlog.get().port);
-				holder.push_back(addr);
+				state std::string addr = "exclude=" + formatIpPort(self->tlog.get().ip, self->tlog.get().port);
 				tokens.push_back(StringRef(addr));
 				ConfigurationResult r =
```

Comment:
From the definition of `changeConfig`, it seems there is an overload possibly simplify this situation:

```
Future<ConfigurationResult> changeConfig(Reference<DB> db, std::string const& modes, bool force)
```

Reply:
The comment of this interface:
```
 Accepts tokens separated by spaces in a single string
```

Reply:
Good idea! Switch to that API call. PTAL!